### ISSUE TITLE
Update to jCal draft 02 (rrule, geo, request-status)

### ISSUE THREAD

**Issue by kewisch** [kewisch/ical.js#79]
The draft is not public yet, but if possible I'd like to add support for it so I can mention this implementation in the draft. Notable changes are:

``` json
["geo", {}, "float", [123.456, 123.456]],
["request-status", {}, "text", ["3.1", "Invalid property value", "DTSTART:96-Apr-01"]],
["rrule", {}, "recur", { "freq": "MONTHLY", "byday": ["MO","-1TU","3WE"], "bymonthday": [2,3,4] }],
```

Especially the RRULE part will probably be very easy to do, we just have to move the fromString code from recur.js to design.js. I have part of this already done.

Some questions that have come up regarding the recur type:
- Should we still be keeping the separation between parts and options?
- Does gaia need compatibility so that it also understands a string value? Any other sort of compatibility?
- Do we still need the fromString method? It would probably be nice for consistency, but I see no direct use for it anymore.

Personally I'm not too happy about special casing geo/request-status, but this was requested and I guess I can see a use for it. Its just kind of weird that the text/float value has an array as a value.

**Comment by lightsofapollo** [comment]
This should be fairly simple... we can add an option in design to handle these cases (like for multiValued props now).

Gaia parses a ICAL string each time (which is fairly fast) which makes jCal changes irrelevant. That may change once we fully formalize the spec.

**Comment by kewisch** [comment]
GEO and REQUEST-STATUS is still under discussion

**Comment by kewisch** [comment]
Fixed by #106
